proxy/envoy/config.go

```diff
@@ -607,13 +612,40 @@ func buildClusters(services []*model.Service) []Cluster {
 				Ports:    []*model.Port{port},
 				Tags:     svc.Tags,
 			}
+
 			cluster := Cluster{
 				Name:             OutboundClusterPrefix + clusterSvc.String(),
 				ServiceName:      clusterSvc.String(),
 				Type:             "sds",
 				LbType:           DefaultLbType,
 				ConnectTimeoutMs: DefaultTimeoutMs,
 			}
+
+			if up, _ := upstreamsByHostname[svc.Hostname]; up != nil {
+				cluster.LbType = loadBalanceType(up.GetLbPolicy())
```

Comment:
remove please. @kyessenov has tackled this in #118 (which might serve as a reference point as well).